packages/rocketchat-ui/client/views/app/createChannel.js

```diff
@@ -146,6 +146,10 @@ Template.createChannel.events({
 		const {username} = Blaze.getData(target);
 		t.selectedUsers.set(t.selectedUsers.get().filter(user => user.username !== username));
 	},
+	'click .rc-icon'(e, t) {
```

Comment:
I think you can be more specific here and use `.rc-tags__tag-icon`. There's a lot of `rc-icon` elements on this template.